continuous-pipe.yml.erb

```diff
@@ -5,6 +5,9 @@ variables:
     name: CP_ENVIRONMENT
     expression: '"<%= config.name %>" ~ code_reference.branch'
   CP_ENVIRONMENT_DOMAIN_SUFFIX: -<%= config.name %>.webpipeline.net
+  SENDMAIL_RELAY_HOST:
+    name: SENDMAIL_RELAY_HOST
+    expression: '"mailcatcher.<%= config.name %>-" ~ slugify(code_reference.branch) ~ ".svc.cluster.local"'
```

Comment:
@kierenevans btw, are you definitely sure slugify is added globally now? I remember it previously only was added to the endpoint host name.

If it's everywhere, we can instead use ${CP_ENVIRONMENT}, and slugify that, which will be more useful